google/ggrc-core
# GGRC-4484 Global Reader with No Role in scope of WF missed Read permissions

# Issue description

Collection GET from next url doesn't work : `/api/task_group_tasks?id__in={id}&_={time_stamp}`

# Steps to test the changes

Steps to reproduce:
1. Create a WF with repeat off, create at least one task group task in a task group. Activate WF.
2. Log in as a Global Reader.
3. Go to a WF (Global Reader should NOT have any WF role in this workflow: WF manager, WF member or Task Assignee).
4. Go to 'Task Group tab' tab.
5. Open task group
**Actual Result:** "Uncaught TypeError: Cannot read property 'indexOf' of undefined" error occurs if Global Reader opens Task Group Info panel in Setup tab
**Expected Result:** no errors should be shown up

**NOTE:** If Global Reader is a task group assignee in workflow, the issue reproduced as well.

# Solution description

Added Global Reader's Read permissions for Workflow related objects.

# Sanity checklist




# PR Review checklist
